docs/configuration.asciidoc

```diff
@@ -36,20 +36,22 @@ With this you can use any configuration source that you configured on the `IConf
 
 NOTE: By simply calling `app.UseElasticApm()` without the overload, the agent will read configurations only from environment variables.
 
+NOTE: For better user experience, `LogLevel` for internal apm logger is used from standard `Logging` section inside `appsettings.json` with `Apm` category name.
```

Comment:
These changes only apply to ASP.NET Core (since we activate the new logger in the `ApmMiddlewareExtension`), so I'd try to make it clear in the docs.

I propose to add this hint to the beginning: `For better user experience in case of ASP.NET Core, `LogLevel` for internal apm …`

Reply:
added